common/model-views.cpp

```diff
@@ -3725,13 +3725,13 @@ namespace rs2
                 configurations::update::allow_rc_firmware,
                 false );
             bool is_rc = ( product_line == RS2_PRODUCT_LINE_D400 ) && allow_rc_firmware;
-            std::string available = get_available_firmware_version( product_line );
+            std::string available_fw_ver = get_available_firmware_version( product_line );
```

Comment:
The way I read it this is the same as `recommended_fw_ver`, and that's the same as `bundled_fw_ver`... Let's understand this better and see if we can simplify